packages/components/select/src/option.vue

```diff
@@ -37,16 +37,22 @@ export default defineComponent({
   componentName: 'ElOption',
 
   props: {
+    /**
+     * @description value of option
+     */
     value: {
       required: true,
       type: [String, Number, Boolean, Object],
     },
+    /**
+     * @description label of option, same as `value` if omitted
+     */
     label: [String, Number],
     created: Boolean,
```

Comment:
This prop doesn't seem to work. 🤔

Reply:
It works in the following files：`select.vue` `useOption.ts` `useSelect.ts`.
It works to [Element Plus Example](https://element-plus.org/en-US/component/select.html#create-new-items), But this example has bug(see: https://github.com/element-plus/element-plus/pull/12635). If not set `created`, the select component will not automatically focus on the newly created option.